hyperdome/common/onion.py

```diff
@@ -174,7 +175,7 @@ def __init__(self, common):
             self.tor_geo_ip_file_path,
             self.tor_geo_ipv6_file_path,
             self.obfs4proxy_file_path,
-        ) = self.common.get_tor_paths()
+        ) = tor_paths
```

Comment:
I bet if I changed tor_paths to a namedTuple it would make these reassignments unecessary